src/Illuminate/Database/Query/Builder.php

```diff
@@ -2438,7 +2438,7 @@ protected function removeExistingOrdersFor($column)
     /**
      * Add a union statement to the query.
      *
-     * @param  \Illuminate\Database\Query\Builder|\Closure  $query
+     * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder|\Closure  $query
```

Comment:
This is not correct, for the reason @GrahamCampbell mentioned:

> This is not necessary. Including subtypes of types already in the union adds no extra information.

Reply:
@nunomaduro 
[`Illuminate\Database\Eloquent\Builder`](https://github.com/illuminate/database/blob/v9.12.2/Query/Builder.php#L29) and [`Illuminate\Database\Eloquent\Builder`](https://github.com/illuminate/database/blob/v9.12.2/Query/Builder.php#L29) are not subclassed to each other, so they are not what @GrahamCampbell calls `Person|Man`.

The code that @fagai exemplifies in the issue is valid code that actually works.

＊　https://phpsandbox.io/n/odd-queen-wkk4-prgxg?files=%2Froutes%2Fweb.php

Is there a reasonable reason to exclude `Illuminate\Database\Eloquent\Builder` from PHPDoc?